docs/datalayer.md

```diff
@@ -56,3 +56,53 @@ $happn._mesh.datalayer.events.on('detatch', function(ev) {
 });
 
 ```
+
+### Datalayer Compaction
+
+The datalayer can be set to compact at an interval, or can also be compacted by a call:
+
+*the following code demonstrates an interval compaction configuration:*
+```javascript
+var config = {
+  datalayer: {
+    port: 55007,
+    filename:test_file_interval,
+    compactInterval:10000//compact every 10 seconds
```

Comment:
ah, this fixes it, so let's stick with milliseconds: https://www.npmjs.com/package/milliseconds

```
var ms = require('milliseconds');
 
ms.seconds(2); // 2000 
ms.minutes(2); // 120000 
ms.hours(2);   // 7200000 
ms.days(2);    // 172800000 
ms.weeks(2);   // 1209600000 
ms.months(2);  // 5259600000 
ms.years(2);   // 63115200000 
```